src/gp-angular.js

```diff
@@ -559,21 +559,22 @@ module.provider('GlobalizationPipelineService', [ function() {
  *  <span gp-translate="aKey"></span>
  *  <span gp-translate="aKey" lang="es"></span>
  */
-module.directive('gpTranslate', ['$log', 'GlobalizationPipelineService', function($log, gp) {
+module.directive('gpTranslate', ['$log', '$interpolate', 'GlobalizationPipelineService', function($log, $interpolate, gp) {

```

Comment:
Does `$interpolate` need to be "enabled" anywhere as a module? And, are there docs we can point to with the full syntax?